src/PathParser.ts

```diff
@@ -44,7 +44,7 @@ export default class PathParser extends SVGPathData {
 		super(
 			path
 				// Fix spaces after signs.
-				.replace(/[+-.]\s+/g, '-')
+				.replace(/[+\-.]\s+/g, '-')
```

Comment:
```suggestion
				.replace(/([+\-.])\s+/g, '$1')
```

?

Reply:
```suggestion
				.replace(/([+-.])\s+/gm, '$1')
```
In case of trimming white spaces I guess such expression could be better. What do you think?

Reply:
yep